.bazelrc

```diff
@@ -1,7 +1,12 @@
 build --java_language_version=17 --java_runtime_version=17
 
-# delete testdata package needed for bazel integration tests
-build --deleted_packages=//aspect/testing/tests/src/com/google/idea/blaze/aspect/integration/testdata
-query --deleted_packages=//aspect/testing/tests/src/com/google/idea/blaze/aspect/integration/testdata
+# Delete test data packages, needed for bazel integration tests. Update by running the following command:
+# bazel run @rules_bazel_integration_test//tools:update_deleted_packages
+build --deleted_packages=aspect/testing/tests/src/com/google/idea/blaze/aspect/integration/testdata,clwb/tests/projects/simple/main
+query --deleted_packages=aspect/testing/tests/src/com/google/idea/blaze/aspect/integration/testdata,clwb/tests/projects/simple/main
 
 common --enable_bzlmod
+
+# Required for CLion integration tests on windows
+startup --windows_enable_symlinks
+build --enable_runfiles
```

Comment:
This could also be moved to `.bazelci/clion.yaml`, but these flags are recommend by the [documentation](https://bazel.build/configure/windows#symlink) anyway. So it might be helpful to leave them there.